doc/debug-support.md

```diff
@@ -23,9 +48,14 @@ To get started, create a `ros`-type debug configuration with an `attach` request
 3. To use VS Code's C++ extension with MSVC on Windows, please make sure the VS Code instance is launched from a Visual Studio command prompt.
```

Comment:
Not related to this change, but I think we may want to change the recommendation to use `-DCMAKE_BUILD_TYPE=RelWithDebInfo` because `Debug` configuration for MSVC will ask the toolchain to link against debug version of VC runtime and most likely it won't be able to run with the `Release` version binaries we provided.

Reply:
thanks for the explanation! changing the example to use `RelWithDebInfo`